packages/main/src/components/DynamicPage/DynamicPage.test.tsx

```diff
@@ -0,0 +1,276 @@
+            </FormItem>
+            <FormItem label="Twitter">
+              <Text>@DeniseSmith</Text>
+            </FormItem>
+          </FormGroup>
+        </Form>
+        <Form style={{ paddingBottom: '2rem' }} columnsXL={4} columnsL={4}>
+          <FormGroup title="Addresses">
+            <FormItem label="Home Address">
+              <Text>2096 Mission Street</Text>
+            </FormItem>
+            <FormItem label="Mailing Address">
+              <Text>PO Box 32114</Text>
+            </FormItem>
+          </FormGroup>
+          <FormGroup title="Mailing Address">
+            <FormItem label="Work">
+              <Text>DeniseSmith@sap.com</Text>
+            </FormItem>
+          </FormGroup>
+        </Form>
+      </DynamicPageContentArea>
+    }
+  />
+);
+
+const renderComponentWithoutContent = () => (
+  <DynamicPage
+    title={
+      <DynamicPageTitle
+        headerVisible={true}
+        actions={[
+          <Button design={ButtonDesign.Emphasized}>Edit</Button>,
+          <Button design={ButtonDesign.Transparent}>Delete</Button>,
+          <Button design={ButtonDesign.Transparent}>Copy</Button>,
+          <Button design={ButtonDesign.Transparent}>Toggle Footer</Button>,
+          <Button icon="action" design={ButtonDesign.Transparent}></Button>
+        ]}
+        navigationActions={[
+          <Button icon="full-screen" design={ButtonDesign.Transparent} />,
+          <Button icon="exit-full-screen" design={ButtonDesign.Transparent} />,
+          <Button icon="decline" design={ButtonDesign.Transparent} />
+        ]}
+        breadcrumbs={
+          <Breadcrumbs>
+            <Link>Home</Link>
+            <Link>Page 1</Link>
+            <Link>Page 2</Link>
+            <Link>Page 3</Link>
+            <Link>Page 4</Link>
+            <Link>Page 5</Link>
+          </Breadcrumbs>
+        }
+        heading={<Title>Header Title</Title>}
+        subHeading={<Label>This is a subheading</Label>}
+      >
+        <Badge>Status: OK</Badge>
+      </DynamicPageTitle>
+    }
+    header={
+      <DynamicPageHeader
+        headerContent={
+          <>
+            <div style={{ display: 'flex', flexDirection: 'column' }}>
+              <Label>Location: Warehouse A</Label>
+              <Label>Halway: 23L</Label>
+              <Label>Rack: 34</Label>
+            </div>
+            <div style={{ display: 'flex', flexDirection: 'column', marginLeft: '2vw' }}>
+              <Label>Availability:</Label>
+              <ObjectStatus state={ValueState.Success}>In Stock</ObjectStatus>
+            </div>
+          </>
+        }
+      />
+    }
+  />
+);
+
+describe('DynamicPage', () => {
+  test('with content', () => {
+    const wrapper = mount(renderComponent());
+    expect(wrapper.render()).toMatchSnapshot();
+  });
+  test('without content', () => {
+    const wrapper = mount(renderComponentWithoutContent());
+    expect(wrapper.render()).toMatchSnapshot();
+  });
+});
```

Comment:
can you use react testing library (render, match asFragment())? We want to remove enyzme in the long run